### PR TITLE
New Project Icons

### PR DESCRIPTION
**Pull request by williankrause**
**Backwards Compatibility Implications**

_None_

**New Features**

- IconBlocked
- IconComplete
- IconEpic
- IconInProgress
- IconNotStarted
- IconStory
- IconTask

**Bug Fixes**

- Refined IconNotification, IconRequired SVGs

### CODE REVIEW

File: src/components/Icon/IconNotification.js [hunk 1]
```diff
@@ -31,11 +31,11 @@ const IconNotification = ({
...
-            <g transform="translate(3 3)">
-                <path d="M9 20.5C2.7 20.5-2.5 15.3-2.5 9S2.7-2.5 9-2.5 20.5 2.6 20.5 9c0 6.3-5.2 11.5-11.5 11.5zm0-21C3.8-.5-.5 3.8-.5 9s4.3 9.5 9.5 9.5 9.5-4.3 9.5-9.5S14.2-.5 9-.5z" />
-                <path d="M7.7 3.8h2.6v6.5H7.7z" />
-                <path d="M7.7 11.6h2.6v2.6H7.7z" />
-            </g>
+            <path d="M12 23.5C5.7 23.5.5 18.3.5 12S5.7.5 12 .5 23.5 5.6 23.5 12c0
+                6.3-5.2 11.5-11.5 11.5zm0-21c-5.2 0-9.5 4.3-9.5 9.5s4.3 9.5 9.5
+                9.5 9.5-4.3 9.5-9.5-4.3-9.5-9.5-9.5z"
+            />
+            <path d="M10.5 6h3v7.5h-3V6zm0 9h3v3h-3v-3z" />
```
Comment (jfrej) [inline src/components/Icon/IconNotification.js]: @mattpauldavies , is this just to optimise the SVG?

I think you should put a note in the changelog (PR description) for this. If it causes any problems in the future, it'll be easier to find the right PR and commit.

File: src/components/Icon/IconRequired.js [hunk 2]
```diff
@@ -31,22 +31,10 @@ const IconRequired = ({
...
-                    <g transform="matrix(0 -1 -1 0 4.5 8.5)">
-                        <path transform="rotate(-90 313.64 631.612)" d="M312.2 633h2.8v-2.8h-2.8z" />
-                        <path transform="rotate(-90 333.157 631.612)" d="M331.8 633h2.8v-2.8h-2.8z" />
-                    </g>
-                </g>
-            </g>
+            <path d="M2 16.4l1.4 2.5 2.3-1.4L4.4 15zM22 7.7l-1.3-2.4-2.4 1.4 1.4 2.4zM5.7
+              6.7L3.3 5.3 2 7.7l2.3 1.4zM18.3 17.5l2.3 1.4 1.4-2.5-2.4-1.4zM10.7
+              20.1h2.9V23h-2.9zM10.7 1h2.9v2.9h-2.9z"
+            />
```
Comment (jfrej) [inline src/components/Icon/IconRequired.js]: Same here, this needs to be mentioned in the changelog. Probably just a generic note about adjusting existing icon SVG will do.

File: src/components/Icon/IconStory.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        >
+            <title id={iconTitleId}>{title}</title>
+            <rect x="13.2" y="6.2" className="uir-icon-line-a" width="8.5" height="1.5" />
+            <rect x="2" y="13.2" className="uir-icon-line-b" width="20" height="1.5" />
+            <rect x="2" y="19.2" className="uir-icon-line-c" width="20" height="1.5" />
+            <polygon
+                className="uir-icon-shape"
+                points="9,4.9 9,3.5 2,3.5 2,4.9 2,6.3 2,7.7
+                7.6,7.7 7.6,9.1 2,9.1 2,10.5 9,10.5 9,9.1
+                9,7.7 9,6.3 3.4,6.3 3.4,4.9"
```
Comment (jfrej) [inline src/components/Icon/IconStory.js]: Are you sure it's safe to have line breaks here? It's something we should probably test in all target browsers.

File: src/components/Icon/IconStory.story.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        `
+SVG story icon
+        `,
+        <IconStory />,
+        <div>
+            <IconStory priority="none" />
+            <IconStory priority="low" />
+            <IconStory priority="medium" />
+            <IconStory priority="high" />
+            <IconStory isBlocked />
```
Comment (jfrej) [inline src/components/Icon/IconStory.story.js]: Could you add an example of an icon with both the `priority` and the `isBlocked` props set?

File: src/components/Icon/IconTask.story.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        `
+SVG task icon
+        `,
+        <IconTask />,
+        <div>
+            <IconTask priority="none" />
+            <IconTask priority="low" />
+            <IconTask priority="medium" />
+            <IconTask priority="high" />
+            <IconTask isBlocked />
```
Comment (jfrej) [inline src/components/Icon/IconTask.story.js]: Same here, could you add an example with both `priority` and `isBlocked`?

File: src/components/Icon/IconNotification.js (hunk 1 above)
Comment (mattpauldavies) [reply src/components/Icon/IconNotification.js]: Yeah I noted it was blurry, on closer inspection the exclamation point was on subpixels.  Will add a note to the description 👍

File: src/components/Icon/IconStory.js (hunk 3 above)
Comment (mattpauldavies) [reply src/components/Icon/IconStory.js]: I believe it is... although to be extra safe, let's whip them out

**Pull request merged**